lib/Backend/Inline.cpp

```diff
@@ -1928,7 +1950,7 @@ Inline::InlineBuiltInFunction(IR::Instr *callInstr, const FunctionJITTimeInfo *
     IR::Instr *inlineBuiltInEndInstr = nullptr;
     if (inlineCallOpCode ==  Js::OpCode::InlineFunctionApply)
     {
-       inlineBuiltInEndInstr = InlineApply(callInstr, inlineeData, inlinerData, symCallerThis, pIsInlined, profileId, recursiveInlineDepth);
+       inlineBuiltInEndInstr = InlineApply(callInstr, inlineeData, inlinerData, symCallerThis, pIsInlined, profileId, recursiveInlineDepth, inlineCallArgCount - 1);
```

Comment:
So should the above assert be changed to inlineCallArgCount >=1 ?

---
In reply to: [85823517](https://github.com/Microsoft/ChakraCore/pull/1838#discussion_r85823517) [](ancestors = 85823517,85817969)